recipes/kwant/meta.yaml

```diff
@@ -0,0 +1,83 @@
+{% set name = "kwant" %}
+{% set py3k_ver = "1.2.2" %}
+{% set py2k_ver = "1.1.2" %}
+{% set variant = "openblas" %}
+
+package:
+  name: {{ name }}
+  version: {{ py3k_ver }}  # [py3k]
+  version: {{ py2k_ver }}  # [py2k]
+
+source:
+  fn: v{{ py2k_ver }}.tar.gz  # [py2k]
+  fn: v{{ py3k_ver }}.tar.gz  # [py3k]
+  url: https://gitlab.kwant-project.org/kwant/kwant/repository/archive.tar.gz?ref=v{{ py2k_ver }}  # [py2k]
+  url: https://gitlab.kwant-project.org/kwant/kwant/repository/archive.tar.gz?ref=v{{ py3k_ver }}  # [py3k]
+  sha256: d298791b920aa5a8191a83178e85472310db678b8237eaa928a8b2a347e825d8  # [py3k]
+  sha256: a47544a8416b19a56da3d511610c9e09ed3bf84f66461d9f95f0f6fefebd0179  # [py2k]
+
+build:
+  skip: true  # [win]
+  number: 200
+  features:
+    - blas_{{ variant }}
+
+requirements:
+  build:
+    - python
+    - toolchain
+    - gcc
+    - blas 1.1 {{ variant }}
+    - openblas  0.2.18|0.2.18.*
+    - mumps
+    - numpy
+    - scipy
+    - cython
+    - nose
+    - tinyarray
+
+  run:
+    - python
+    - blas 1.1 {{ variant }}
+    - openblas 0.2.18|0.2.18.*
+    - mumps
+    - numpy
+    - matplotlib
+    - scipy
+    - cython
+    - nose
+    - tinyarray
+    - libgfortran
+    - libgcc
```

Comment:
I see. So it does need C++.

Please add `yum_requirements.txt` with `devtoolset-2-gcc-gfortran`. Also please add a selector to use `gcc` on OS X only. Then please add `export LIBRARY_PATH="${PREFIX}/lib"` to `build.sh`. Finally please drop `libgcc`, but keep/add `libgfortran` (in `build` and `run`).